src/shogun/kernel/MaternKernel.cpp

```diff
@@ -0,0 +1,170 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Gil Hoben
+ */
+
+#include <shogun/distance/EuclideanDistance.h>
+#include <shogun/kernel/MaternKernel.h>
+
+using namespace shogun;
+
+
+MaternKernel::MaternKernel(): ShiftInvariantKernel()
+{
+	register_params();
+}
+
+MaternKernel::MaternKernel(int32_t size, float64_t w, float64_t order): 
+	ShiftInvariantKernel(), m_nu(order), m_width(w)
```

Comment:
could you call `MaternKernel` here to avoid having to call register params again?

Reply:
Yup, need to clean this up